spec/system/comments_spec.rb

```diff
@@ -480,64 +564,64 @@
       expect(page).not_to have_css "#js-comment-form-comment_#{comment.id}"
     end
 
-    scenario "can not comment as an administrator" do
-      moderator = create(:moderator)
-
-      login_as(moderator.user)
-      visit debate_path(debate)
+    scenario "cannot comment as an administrator" do
+      visit polymorphic_path(resource)
 
       expect(page).not_to have_content "Comment as administrator"
     end
   end
 
   describe "Administrators" do
-    scenario "can create comment as an administrator" do
-      admin = create(:administrator)
-
-      login_as(admin.user)
-      visit debate_path(debate)
-
-      fill_in "Leave your comment", with: "I am your Admin!"
-      check "comment-as-administrator-debate_#{debate.id}"
-      click_button "Publish comment"
-
-      within "#comments" do
-        expect(page).to have_content "I am your Admin!"
-        expect(page).to have_content "Administrator ##{admin.id}"
-        expect(page).to have_css "div.is-admin"
-        expect(page).to have_css "img.admin-avatar"
+    context "comment as administrator" do
+      scenario "can create comment" do
+        admin = create(:administrator, description: "valuator user")
```

Comment:
When first reading it, I thought we would be in the valuation section :thinking:. How about:

```suggestion
        admin = create(:administrator, description: "admin user")
```